src/modules/main/components/Header/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from "react-router-dom";
+
+import './Header.css';
+
+import MainView from '../MainView';
+import DirectionView from "../../../direction/components/DirectionView";
+
+
+function Header() {
+  return (
+    <div className="header">
+
+      <div className="general">
+        <div className="logo">  
+          <img src="#" alt="logo" className="logo-img"/>
+        </div>
+        <div className="log-in">
+          <img src="#" alt="user" className="img-log-in"/>
+          <button type="button">Log in</button>
+          <button type="button">Register</button>
+        </div>  
+      </div>
+
+      <div className="navigation">
+        <ul className="nav-list">
+          <li><Link to="/">Головна</Link></li>
+          <li><Link to="/covid-19">Covid-19</Link></li>
```

Comment:
please use only valid paths that we have right not: "/" and "covid-19", for others - just use `li` elements